polls/views.py

```diff
@@ -36,9 +36,9 @@ class ResultsView(generic.DetailView):
 
 
 def vote(request, question_id):
-    question = get_object_or_404(Question, pk=question_id)
+    question: Question = get_object_or_404(Question, pk=question_id)
```

Comment:
I believe this is your only usage of type annotations in the project. Was it necessary, to get PyCharm to shut up?